src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java

```diff
@@ -65,8 +65,12 @@
         + "\n* Confirm service online within 21 days of sending the form, particulars and response pack, before"
         + " 4pm if you're doing this on the due day";
 
-    public static final String LIP_CONFIRMATION_BODY = "<br />You do not need to do anything.\n\n"
-        + "Your claim will be considered by the court and you will be informed of the outcome by post.";
+    public static final String LIP_CONFIRMATION_BODY = "<br />To continue your claim by post you need to:"
+        + "\n* [Download the sealed claim form](%s)"
+        + "\n* Send the claim form, <a href=\"%s\" target=\"_blank\">a response pack</a> (PDF, 266 KB) "
```

Comment:
(PDF, 266 KB) does it need to hard coded

Reply:
good question, it's currently hardcoded for other confirmation body. Will ask Steve in ticket